cmd/create/idp/openid.go

```diff
@@ -49,8 +49,6 @@ func buildOpenidIdp(cmd *cobra.Command,
 	}
 
 	if interactive.Enabled() {
-		instructionsURL := "https://docs.openshift.com/dedicated/identity_providers/" +
```

Comment:
This is guiding towards openID and the other is towards ldap, they are different :)

Reply:
Yikes - since the line length linter broke these up into separate "constants" it flagged the two, when only the first half was shared :facep

Reply:
(I'd personally suggest just removing the `goconst` linter, but that's a bigger change than obeying it for now)